vector/src/main/java/im/vector/app/features/home/HomeActivity.kt

```diff
@@ -697,6 +700,12 @@ class HomeActivity :
         }
     }
 
+    private fun inviteByEmail() {
+        InviteByEmailDialog(this).show { email ->
+            sharedActionViewModel.post(HomeActivitySharedAction.InviteByEmail(email))
```

Comment:
The invite by email is not working on my side. I did some tests and this event is not observed anymore, so the action is not handled.

The old layout was using the `HomeDetailFragment`, the `sharedActionViewModel` is observed there and call the `createRoomViewModel` to create the DM by email.
The new layout use `NewHomeDetailFragment`, the previous mechanism should be moved/duplicated in this fragment.

Another solution would be to do in the `HomeActivity` instead, there is already an observation of the event there but there is not action on it because the event was already handled in the fragment.